packages/orchestration/src/orchestration-api.ts

```diff
@@ -88,6 +89,8 @@ export interface Chain<CI extends ChainInfo> {
   makeAccount: () => Promise<OrchestrationAccount<CI>>;
   // FUTURE supply optional port object; also fetch port object
 
+  query: CI extends { icqEnabled: true } ? ICQQueryFunction : never;
```

Comment:
`never` is maybe not correct here since all chain objects will have `.query()`. If it's called and `icqEnabled = false`, it will throw

Reply:
This looks right to me. It's saying that if `icqEnabled` then `.query` should have type `never` and so never be used.